release/smoke-tests/run-tarball-files-smoke-tests.sh

```diff
@@ -146,12 +142,14 @@ function run_smoke_test() {
     cd "${CURRENT_DIR}" || exit
 }
 
-while getopts "b:d:hn:u:v:" arg; do
+while getopts "b:d:hi:n:t:u:v:" arg; do
```

Comment:
Should `-i <string>` and `-t <string>` be a required argument? On lines 194-202 it looks like FROM_IMAGE and TAR_NAME have default values.

Reply:
No, I don't think they need to be required. Is this requiring them?

Reply:
I have been able to run this script without `-i` and `-t`.

Reply:
Thanks for confirming `-i` and `-t` are actually optional. Checking the `getops` documentation confirms an option character followed by a `:` is expecting an argument if the flag is present.

If curious, the official documentation is available here:
https://man7.org/linux/man-pages/man1/getopts.1p.html